### DISCUSSION TITLE
Can Zitadel be used as an identity provider for other applications?

### DISCUSSION THREAD

**Discussion by osdiab** [zitadel/zitadel#6051] [Q&A] [answered]
As the title describes - I don't need it now but while I understand that with the Identity Brokering concept that other IdPs can be used to log into an application secured by Zitadel, could other applications in return use Zitadel as an IdP to log into third party apps? I am having a bit of hard time understanding from the marketing page/docs if this is the case, would be good to know before making the choice. Thanks!

**Comment by hifabienne** [comment, marked as answer]
Hei @osdiab 
Yes this is possible currently with the standard OIDC and SAML.
You can find some guides for integrations on our documentation page: https://zitadel.com/docs/guides/integrate/services
OIDC is the easiest and recommended way to integrate.

**Comment by osdiab** [comment]
Great, thanks!

Omar
